.github/actions/create-jira-issues-for-community-activities/action.yml

```diff
@@ -0,0 +1,103 @@
+﻿name: Create Jira issues for community activities
+description: >
+  Creates Jira issues for community activities happening on GitHub, like issues, discussions, pull requests being
+  opened.
+
+inputs:
+  issue-component:
+    required: false
+    description: Optional component of the issues being created.
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Scripts
+      shell: pwsh
+      run: |
+        "${{ github.action_path }}" >> $Env:GITHUB_PATH
+        cd ${{ github.action_path }}
+        cd ../
+        "$((Get-Location).Path)/verify-submodule-pull-request" >> $Env:GITHUB_PATH
```

Comment:
Why not join the path instead?
`Join-Path -Resolve "${{ github.action_path }}" .. verify-submodule-pull-request`